### DISCUSSION TITLE
Showing the rust-analyzer warnings

### DISCUSSION THREAD

**Discussion by blitzerr** [helix-editor/helix#5233] [General] [answered]
Helix shows the code errors but when it comes to code warnings, it shows as a yellow spot in the gutters but when I go to that line, it does not show what the warning is. Is there a way to enable it ?

**Comment by the-mikedavis** [comment, marked as answer]
See https://github.com/helix-editor/helix/issues/3078: the diagnostic text shows up when the cursor is within the diagnostic's range rather than on the same line as the diagnostic. You can use `]d` to jump to the next diagnostic range and `[d` to jump to the previous.

**Reply by blitzerr** [reply]
Thanks a lot @the-mikedavis . I am enjoying helix a lot.
